sdk/samples/azidentity/example_SDKV1_test.go

```diff
@@ -0,0 +1,133 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package azidentitysamples
+
+import (
+	"context"
+	"fmt"
+	"os"
+
+	"github.com/Azure/azure-sdk-for-go/sdk/azcore"
+	"github.com/Azure/azure-sdk-for-go/sdk/azidentity"
+	"github.com/Azure/azure-sdk-for-go/sdk/to"
+	"github.com/Azure/azure-sdk-for-go/services/resources/mgmt/2019-05-01/resources"
+
+	"github.com/jongio/azidext/go/azidext"
+)
+
+// Please note that the examples in this file are using the Azure SDK for Go V1 code base, along
+// with azidentity package from the Azure SDK for Go V2.
+// The adapter in the azidext package provides a simple way to integrate azidentity credentials
+// as authorizers for the V1 code base.
+
+const (
+	groupName = "samplegroup"
+)
+
+// Environment variables required for EnvironmentCredential to work and/or DefaultAzureCredential
+var (
+	clientID       = os.Getenv("AZURE_CLIENT_ID")
+	clientSecret   = os.Getenv("AZURE_CLIENT_SECRET")
+	subscriptionID = os.Getenv("AZURE_SUBSCRIPTION_ID")
+	tenantID       = os.Getenv("AZURE_TENANT_ID")
+)
+
+var (
+	location  = os.Getenv("AZURE_LOCATION")
+	userAgent = "azidentitysample"
+)
+
+// Example for using the DefaultAzureCredential through the NewDefaultAzureCredentialAdapter and assigning the credential to the
+// SDK V1 authorizer.
+func getGroupsClientWithDefaultAzureCredential() resources.GroupsClient {
+	groupsClient := resources.NewGroupsClient(subscriptionID)
+	a, err := azidext.NewDefaultAzureCredentialAdapter(nil)
+	if err != nil {
+		panic("failed to get credential")
+	}
+	groupsClient.Authorizer = a
+	groupsClient.AddToUserAgent(userAgent)
+	return groupsClient
+}
+
+// Example for using the ClientSecretCredential with the NewAzureIdentityCredentialAdapter and assigning the credential to the
+// SDK V1 authorizer.
+func getGroupsClientWithClientSecretCredential() resources.GroupsClient {
+	groupsClient := resources.NewGroupsClient(subscriptionID)
+	cred, err := azidentity.NewClientSecretCredential(tenantID, clientID, clientSecret, nil)
+	if err != nil {
+		panic(err)
+	}
+	a := azidext.NewAzureIdentityCredentialAdapter(cred, azcore.AuthenticationPolicyOptions{Options: azcore.TokenRequestOptions{Scopes: []string{"https://management.azure.com/.default"}}})
```

Comment:
The specification of scopes is going to be new for track 1 users.  I think this needs a comment explaining what this does along with any links to our Azure authentication docs.

We should also think if there's a way to simplify this in the adapter for the common case.

Reply:
Done. Although, on the improvement side of things...I could specify a default endpoint for track1 in this example and have a comment above it. Since track1 is for ARM it could be that we could explain the default scope for track1 should generally be the management scope.